hypha/static_src/src/sass/apply/components/_wrapper.scss

```diff
@@ -202,7 +202,7 @@
         }
 
         .card:first-child {
```

Comment:
I think we can just remove this? 
`.card` is used at other places as well but I don't think its first child top margin makes sense anywhere.